tree/dataframe/CMakeLists.txt

```diff
@@ -103,6 +103,8 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTDataFrame
   DICTIONARY_OPTIONS
     -writeEmptyRootPCM
     ${RDATAFRAME_EXTRA_INCLUDES}
+  LIBRARIES
+    ROOTIOLite
```

Comment:
Shouldn't that come transitively from RIO?

Reply:
Yes, it should but that's not implemented. I had to implement something similar for LLVM. Here is how's done -- https://github.com/root-project/root/pull/6385/files#diff-93558e500b17901f800fbb40a2c2fdfd6ee229077db63413bd4a91ee7ec66ea7R498-R531

Reply:
LLVM is its own project; RIO is part of ROOT and as such this should work without the mention of `ROOTIOLite` for targets that already depend on `RIO`.